package/nfs/rancher-nfs

```diff
@@ -182,5 +183,24 @@ delete() {
     print_success purged
 }
 
+validate() {
+
+    # default configuration
+    local host="$NFS_SERVER"
+    local exportDir="$MOUNT_DIR"
+    local opts="$MOUNT_OPTS"
+    local mountDir="$(tmp_dir)"
+
+    log_info "validation" "Mounting" "$mountDir" "for validating NFS configuration."
```

Comment:
Those separate quotes are unnecessary.

"Validating NFS by mounting to $mountDir" should be enough.

Reply:
The first parameter of function `log_info` needs a name, so I put a `validation` into it. I will make it right.